docs/changes/wcs/13171.feature.rst

```diff
@@ -0,0 +1 @@
+Updated bundled WCSLIB to version 7.10 with a bug fix in ``tabcpy()`` and other enhancements. For a full list of bug fixes see https://www.atnf.csiro.au/people/mcalabre/WCS/CHANGES
```

Comment:
I don't think you can use "feature" for a bugfix backport to LTS. @saimn can confirm.

Reply:
OK. I'll fix that

Reply:
done